src/lib/fs-utils.ts

```diff
@@ -31,12 +35,10 @@ export function writeFileAtomic(
 export function safeRename(src: string, dest: string): Bluebird<void> {
 	return Bluebird.resolve(fs.rename(src, dest))
 		.then(() => fs.open(Path.dirname(dest), 'r'))
-		.tap(fs.fsync)
-		.then(fs.close);
+		.tap((fh) => fh.sync())
+		.then((fh) => fh.close());
 }
 
-export function pathExistsOnHost(p: string): Bluebird<boolean> {
-	return Bluebird.resolve(fs.stat(Path.join(constants.rootMountPoint, p)))
-		.return(true)
-		.catchReturn(ENOENT, false);
+export function existsAsync(p: string): Bluebird<boolean> {
+	return Bluebird.resolve(fs.stat(p)).return(true).catchReturn(ENOENT, false);
```

Comment:
`fs.access` is the recommended way of checking a file exists over fs.stat: https://nodejs.org/docs/latest-v14.x/api/fs.html#fs_fs_stat_path_options_callback
> To check if a file exists without manipulating it afterwards, fs.access() is recommended.

```suggestion
	return Bluebird.resolve(fs.access(p)).return(true).catchReturn(ENOENT, false);
```

Reply:
Sounds good, I'll change this